EventTicketingSystem.ConsoleTest/ServiceProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿using EventTicketingSystem.DataAccess.Extensions;
+using EventTicketingSystem.DataAccess.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EventTicketingSystem.ConsoleTest
+{
+    internal static class ServiceProvider
```

Comment:
Service locator is bad practice. Why do you need it? Espeicially static

Reply:
It is just a test project.